docs/jobs/ai-assistant.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Using OpenFn with AI Assistant
+sidebar_label: Using AI Assistant
+---
+
+:::warning
+
+AI makes mistakes. Always review the AI's response carefully and make sure to
+verify the output. You can help us improve the AI Assistant by providing
```

Comment:
How will they provide feedback?

Reply:
@christad92 how about starting a dedicated "AI" category in the Community where users could share feedback... and where we could announce the feature and related updates.